examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -382,7 +382,13 @@ static void createMqttTask(void) {
 }
 
 static void initMqtt(void) {
-  if (mqttClient.begin(configuration.getMqttBrokerUri())) {
+  String mqttUri = configuration.getMqttBrokerUri();
+  if (mqttUri.isEmpty()) {
+    Serial.println("MQTT is not configured, skip initialize mqtt client");
+    return;
+  }
+
+  if (mqttClient.begin(mqttUri)) {
     Serial.println("Connect to MQTT broker successful");
```

Comment:
Here we could write the past to indicate that it actually worked

"Connected to MQTT broker successfully"